devices/Sen5x/Entities/StringValue.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+using System;
+
+namespace Iot.Device.Sen5x.Entities
+{
+    /// <summary>
+    /// Represents a null-terminated string value read from the sensor.
+    /// </summary>
+    internal class StringValue : AbstractReadEntity
+    {
+        private int _maxLength;
+
+        internal StringValue(int maxLength = 32)
+        {
+            // Internal constructor, because this is a read-only entity.
+            _maxLength = maxLength;
+        }
+
+        internal override int ByteCount => _maxLength / 2 * 3;
+
+        internal override void FromSpanByte(SpanByte data)
+        {
+            Text = string.Empty;
+            for (int i = 0; i < data.Length; i += 3)
```

Comment:
for maintainability, can you please add a little comment on the format used by this sensor?
Because all up, seems like a Encoding.Default.GetString will do the job

Reply:
Encoding.UTF8.GetString() won't work (without additional steps). I've added a comment in the code to explain why.